src/main/java/UseCaseClasses/ScheduleEditor.java

```diff
@@ -24,11 +25,10 @@ public void addEvent(String eventType, String eventName, String eventDay, int ev
             event.setEventStartTime(eventStartTime);
             event.setEventEndTime(eventEndTime);
 
-            HashMap<String, String> Day = schedule.getSchedule().get(eventDay);
-            for (int i = eventStartTime; i < eventEndTime; ++i) {
-                Day.put(String.valueOf(i), eventName);
-            }
-            schedule.getSchedule().put(eventDay, Day);
+            ArrayList<Event> dayevents = schedule.getSchedule().get(eventDay);
+            dayevents.add(event);
+            //TODO: should I create a setter in schedule to set the list of events of a specific day in the schedule class? or can I just use the getter

```

Comment:
for messing with a specific day, it may be worth making a new method that allows you to set the list of events for a specific day

Reply:
ooo yess that's a good idea and I'll make that in schedule right

Reply:
I made a commit to address this: created a setter in schedule and used it in scheduleEditor's addEvent and removeEvent.